plugins/mac-capture/plugin-main.c

```diff
@@ -2,6 +2,10 @@
 
 OBS_DECLARE_MODULE()
 OBS_MODULE_USE_DEFAULT_LOCALE("mac-capture", "en-US")
+MODULE_EXPORT const char *obs_module_description(void)
+{
+	return "MacOS audio input/output and window/display capture";
```

Comment:
I believe "macOS" _always_ has a lowercase "m", regardless of where it is in a sentence.